charts/rancher-backup/files/default-resourceset-contents/fleet.yaml

```diff
@@ -19,6 +19,9 @@
 - apiVersion: "v1"
   kindsRegexp: "^configmaps$"
   namespaceRegexp: "^cattle-fleet-|^fleet-|^cluster-fleet-"
+- apiVersion: "scheduling.k8s.io/v1"
+  kindsRegexp: "^priorityclasses$"
+  namespaceRegexp: "^cattle-fleet-|^fleet-|^cluster-fleet-"
```

Comment:
If my understanding is correct, priority classes are not namespaced, so this line does not help us here.